app/views/liquids/_user_subscription.html.erb

```diff
@@ -0,0 +1,70 @@
+        <span class="crayons-avatar crayons-avatar--l mr-2 ltag__user-subscription-tag__author-profile-image">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__author-profile-image" src=<%= author_profile_image %> <%= "#{author_username} profile image" %>>
+        </span>
+        <span class="crayons-avatar crayons-avatar--l mr-2 ltag__user-subscription-tag__subscriber-profile-image">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__subscriber-profile-image">
+        </span>
+      </div>
+      <div class="ltag__user-subscription-tag__cta-text fs-3xl" id="user-subscription-cta-text"><%= cta_text %></div>
+      <div class="ltag__user-subscription-tag__subscription-area" id="subscription-area">
+        <div class="ltag__user-subscription-tag__signed-out" id="subscription-signed-out">
+          <div class="fs-base" id="logged-out-text">You must first sign in to DEV.</div>
+          <button class="crayons-btn" id="sign-in-btn">Sign In</button>
+        </div>
+        <div class="ltag__user-subscription-tag__signed-in" id="subscription-signed-in">
+          <button class="crayons-btn" id="subscribe-btn">Subscribe</button>
+          <div class="fs-s" id="logged-in-text">
+            You'll subscribe with
+            <span class="ltag__user-subscription-tag__subscriber-email"></span>
+            the email address associated with your DEV account. To use a
+            different email address, you can
+            <a href="/settings">update your email address in Settings</a>.
+          </div>
+        </div>
+        <div class="ltag__user-subscription-tag__apple-auth fs-s" id="subscriber-apple-auth">
+          <button disabled class="crayons-btn" id="subscribe-btn">Subscribe</button>
+          <div class="fs-s" id="apple-auth-message">
+            Hey, there! It looks like when you created your DEV account you signed
+            up with Apple using a private relay email address. If you'd like to
+            subscribe, please
+            <a href="/settings">update your email address in Settings</a>
+            first to a different email address.
+          </div>
+        </div>
+        <div class="ltag__user-subscription-tag__response-message" id="response-message"></div>
+      </div>
+    </div>
+    <div id="user-subscription-confirmation-modal">
+      <div class="crayons-modal crayons-modal--l">
+        <div class="crayons-modal__box">
+          <header class="crayons-modal__box__header">
+            <h2>Confirm subscribe</h2>
+            <button type="button" class="crayons-btn crayons-btn--icon crayons-btn--ghost" id="close-confirmation-modal">
+              <svg width="24" height="24" viewBox="0 0 24 24" class="crayons-icon" xmlns="http://www.w3.org/2000/svg">
+                <path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636l4.95 4.95z" />
```

Comment:
we should run this [through svgo](https://jakearchibald.github.io/svgomg) if we haven't already!

Reply:
👀  I copied and pasted it directly from [Crayons](https://dev.to/page/crayons). So if/when we do this post-design we should update that page as well!